zerver/migrations/0537_customprofilefield_editable_by_user.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.6 on 2024-05-31 02:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0536_add_message_type"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="customprofilefield",
+            name="editable_by_user",
+            field=models.BooleanField(default=True),
```

Comment:
@alexmv should we start using `db_default` for field additions like this?

Reply:
I ... guess?  It makes prod inserts still work after the migration has run.